src/Library.java

```diff
@@ -0,0 +1,186 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Library {
+    private Book[] books = new Book[100];
+    private int bookCount;
+    private Student[] students = new Student[100];
+    private int studentCount;
+
+    private static final String CYAN = "\u001B[36m";
+
+    /**
+     * Adds a new book to the library itself. (Book array)
+     * @param title
+     * @param author
+     * @param isbn
+     */
+    public void addBook(String title, String author, long isbn) {
+        if (bookCount < books.length) {
+            books[bookCount++] = new Book(title, author, isbn);
+        } else {
+            System.out.println("Library is full. Cannot add more books.");
+        }
+    }
+
+    /**
+     * Removes a book from the library.
+     * @param isbn
+     */
+    public void removeBook(long isbn) {
+        for (int i = 0; i < bookCount; i++) {
+            if (books[i].getIsbn() == isbn) {
+                books[i] = books[--bookCount]; // Replace with the last book
+                books[bookCount] = null; // Clear the last position
+                return;
+            }
+        }
+        System.out.println("Book not found.");
+    }
+
+    /**
+     * Finds a book by its ISBN number.
+     * @param isbn
+     */
+    public Book findBookByIsbn(long isbn) {
```

Comment:
you should be storing ISBNs as strings I think, it's weird to treat an ISBN as an actual number.... though I guess it is more memory efficient technically